libraries/AP_HAL_ChibiOS/hwdef/CubeOrangePlus-ODID/defaults.parm

```diff
@@ -0,0 +1,21 @@
+# setup correct defaults for battery monitoring for cube power brick
+BATT2_CURR_PIN 4
+BATT2_VOLT_PIN 13
+BATT_AMP_PERVLT 39.877
+BATT_VOLT_MULT 12.02
+BATT2_AMP_PERVLT 39.877
+BATT2_VOLT_MULT 12.02
+# setup ADSB
+ADSB_TYPE 1
+SERIAL5_BAUD 57
+SERIAL5_PROTOCOL 1
+
+DID_ENABLE 1 @READONLY
+DID_OPTIONS 1 @READONLY
+DID_CANDRIVER 1
+AHRS_EKF_TYPE 3 @READONLY
+GPS1_TYPE 1
+GPS2_TYPE 0
+
+EK2_PRIMARY 1
```

Comment:
Really not sure what these are doing in here.  I can see you've just swiped them in from the other file.... but still odd.